### PR TITLE
U30 soft kernel hello world testcase

### PR DESCRIPTION
**Pull request by sarab96**
U30 soft kernel hello world testcase.
Use make or runit.sh to compile and run

### CODE REVIEW

File: tests/xrt/soft_kernel_hello/Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+RUN_SHELL := $(shell ./runit.sh)
+
+$(info $(RUN_SHELL))
```
Comment (sonals) [inline tests/xrt/soft_kernel_hello/Makefile:4]: Please do not call scripts from Makefile. 
Use xclbin.mk for compiling PL kernels with v++ and CMakeLists.txt for host code. See examples like:
XRT/tests/xrt/13_add_one
XRT/tests/xrt/100_ert_ncu

**Review by sonals — CHANGES_REQUESTED** [review]
Details in email...

File: tests/xrt/soft_kernel_hello/Makefile (hunk 1 above)
Comment (sarab96) [reply tests/xrt/soft_kernel_hello/Makefile:4]: done

**Comment by sarab96** [comment]
Changes done

**Comment by sarab96** [comment]
retest this please

**Comment by sarab96** [comment]
retest this please

**Pull request merged**